Bmx.Core/BmxCore.cs

```diff
@@ -70,7 +62,25 @@ public class BmxCore {
 				// TODO: Remove need for input here, Push style flows have no user input on app here
 				// ex: https://developer.okta.com/docs/reference/api/authn/#response-example-waiting
 				PromptMfaInput( selectedMfa.Name );
+				throw new NotImplementedException();
 			}
+
+
+			// TODO: Decouple this more, there is an implicit understanding its an Okta AWS app
```

Comment:
Ah because users can manually add their own login that isn't SAML?

Reply:
Could you elaborate on this, my understanding is current AWS apps within Okta will all follow the same convention (in terms of API response). Is it possible for users to manually add an AWS account on Okta? 

This todo was more in the context that I need to move the "amazon_aws" string out of Bmx Core into a app settings.

Reply:
> This todo was more in the context that I need to move the "amazon_aws" string out of Bmx Core into a app settings.

Ah gotcha

Sometimes, Okta offers to save username / password pairs for you as you sign into stuff (e.g. LMS). I thought you were referring to someone that added an AWS account manually. It's an edge case, and maybe it already gets filtered out by the time the code is here?

Reply:
Did a few tests, as far as I can tell those do not get saved as "amazon_aws" in Okta. 

However I'm not 100% sure on these results, it does seem like an edge case. My understanding isn't supposed BMX to handle manually added credentials, requires further investigation.